Scripts/LineageOS-17.1/Patch.sh

```diff
@@ -539,14 +539,6 @@ if enterAndClear "device/oppo/common"; then
 awk -i inplace '!/TARGET_RELEASETOOLS_EXTENSIONS/' BoardConfigCommon.mk; #disable releasetools to fix delta ota generation
 fi;
 
-if enterAndClear "device/xiaomi/davinci"; then
-smallerSystem;
```

Comment:
are you sure it fits?

Reply:
The system partition in davinci is 3.5 GiB large, so I don't see why the system image would need further reduction. I have used DivestOS in davinci in the past up to 19.1, without using smallerSystem, and I haven't faced any issues.